tests/providers/google/cloud/operators/test_automl.py

```diff
@@ -21,11 +21,16 @@
 from unittest import mock
 
 import pytest
-from google.api_core.gapic_v1.method import DEFAULT
-from google.cloud.automl_v1beta1 import BatchPredictResult, Dataset, Model, PredictResponse
 
-from airflow.providers.google.cloud.hooks.automl import CloudAutoMLHook
-from airflow.providers.google.cloud.operators.automl import (
+# For no Pydantic environment, we need to skip the tests
+pytest.importorskip("google.cloud.aiplatform_v1")
+
+from google.api_core.gapic_v1.method import DEFAULT  # noqa: E402
+from google.cloud.automl_v1beta1 import BatchPredictResult, Dataset, Model, PredictResponse  # noqa: E402
+
+from airflow.exceptions import AirflowProviderDeprecationWarning  # noqa: E402
+from airflow.providers.google.cloud.hooks.automl import CloudAutoMLHook  # noqa: E402
+from airflow.providers.google.cloud.operators.automl import (  # noqa: E402
```

Comment:
Instead of having these `#noqa: E402`, could we maybe add a line in `pyproject.toml` (around L356)

Reply:
I was not aware of this feature. Thank you for the information! I am changing accordingly